code/modules/food/cooking_machines.dm

```diff
@@ -62,7 +62,7 @@ var/global/ingredientLimit = 10
 // Base (Oven) /////////////////////////////////////////////////
 
 /obj/machinery/cooking
-	name = "oven"
+	name = "Waffle Inc. Ez-bake FUN oven"
```

Comment:
unatomic

Reply:
Needed to differentiate the two, else they'd share the same sprites and the same name.